Refresher/UI/PSPSetupForm.cs

```diff
@@ -26,18 +26,38 @@ public PSPSetupForm() : base("PSP Setup")
         
         foreach (DriveInfo drive in drives)
         {
+            bool isVita = false;
+            string pspEmuFolderName = "";
+
             try
             {
                 Console.WriteLine($"Checking drive {drive.Name}...");
-                
-                //If theres no PSP folder,
-                if (!Directory.Exists(Path.Combine(drive.RootDirectory.FullName, "PSP")))
+
+                // Match for all directories called PSP and PSPEMU
+                // NOTE: we do this because the PSP filesystem is case insensitive, and the .NET STL is case sensitive on linux
+                IEnumerable<string> possiblePspMatches = Directory.EnumerateDirectories(drive.RootDirectory.FullName, "PSP", new EnumerationOptions
```

Comment:
Can probably just use `GetDirectories`() and work with an array directly

Reply:
.NET doesn't provide a way to check for a single directory while ignoring case sensitivity, which makes usage of `EnumerateDirectories` valid. Annoying, but it works.